jbmc/regression/jbmc/stub_identifiers/test.desc

```diff
@@ -0,0 +1,18 @@
+CORE symex-driven-lazy-loading-expected-failure
+Test.class
+--show-symbol-table
+^EXIT=0$
+^SIGNAL=0$
+--
+^Symbol\.*: malloc_site
+^Symbol\.*: to_construct
+^Symbol\.*: to_return
+^Symbol\.*: this
+^Symbol\.*: caught_exception
+^Symbol\.*: anonlocal
+^Symbol\.*: stub_ignored_arg
+^Symbol\.*: #return_value
+^Symbol\.*: return_tmp
+^Symbol\.*: new_tmp
+^Symbol\.*: tmp_new_data_array
+^Symbol\.*: newarray_tmp
```

Comment:
Add positive conditions too? At the moment `main() { return 0; }` would pass this test-case ;)

Reply:
That would be perfectly fine. I only want to check that no non-prefixed variables are generated - these break goto-diff.